charts/opensearch/templates/securityconfig.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
   name: {{ .Values.securityConfig.config.securityConfigSecret }}
   namespace: {{ .Release.Namespace }}
   labels:
-    app: {{ .Chart.Name }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    app: { { .Chart.Name } }
+    release: { { .Release.Name } }
+    heritage: { { .Release.Service } }
```

Comment:
Weird looking code block here. Can you revert back?

Reply:
My apologies, got lost in the transition between the repos and an overzealous formatter.